source/fundamentals/crud/read-operations/sort.txt

```diff
@@ -4,11 +4,199 @@ Sort Results
 
 .. default-domain:: mongodb
 
-Use ``sort()`` to change the order in which read operations return
-documents. The ``sort()`` instance method tells MongoDB to order returned documents by the
-values of one or more fields in a certain direction. To sort returned
-documents by a field in ascending (lowest first) order, use the static method ``Sorts.ascending()``,
-specifying the field on which to sort. To sort in descending (greatest first) order instead,
-use the static method ``Sorts.descending()``, specifying the field on which to sort. For example,
-you can sort on ``id`` or ``name`` fields. If you do not specify a sort, MongoDB does not
-guarantee the order of query results.
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to specify the order of your results
+from read operations.
+
+Sample Data
+~~~~~~~~~~~
+
+To run the examples in this guide, load these documents into the
+``ratings`` collection of the ``tea`` database with the following
+snippet:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/sort.go
+   :language: go
+   :dedent:
+   :start-after: begin insertDocs
+   :end-before: end insertDocs
+
+.. tip:: Non-existent Database and Collections
+
+   The driver automatically creates the necessary database and/or collection
+   when you perform a write operation against them if they don't already exist.
+
+Each document contains a rating for a type of tea, which corresponds to
+the ``type`` and ``rating`` fields.
+
+.. note::
+
+   Each example truncates the ``ObjectID`` value since the driver
+   generates them uniquely.
+
+Sort Direction
+--------------
+
+To specify the order your results, pass an interface containing the
+field and sort direction in to the ``FindOptions.SetSort()`` function
+as the third parameter to the ``Find()`` function.
+
+The sort direction can be **ascending** or **descending**.
+
+Ascending
+~~~~~~~~~
+
+An ascending sort orders your results from smallest to largest. To
+specify this sort, pass the field you want to sort by and a ``1`` to the
+``FindOptions.SetSort()`` function.
+
+.. tip::
+
+   With an ascending sort, a ``Boolean`` orders ``false`` **then**
+   ``true``, a ``String`` orders from **a to z** and numeric types order
```

Comment:
Change to:
values of type ``Boolean`` are ordered from ``false`` to ``true``, ``String`` type values are ordered from **a to z**, and numeric type values are ordered .....

Reply:
Please check the bold text in html -- it renders strangely on current staging. See if you can get by without bold here, maybe, as I am not sure why it renders the way it does.